cms/envs/common.py

```diff
@@ -465,6 +465,10 @@
 USE_I18N = True
 USE_L10N = True
 
+STATICI18N_ROOT = PROJECT_ROOT / "static"
+
+STATICI18N_OUTPUT_DIR = lms.envs.common.STATICI18N_OUTPUT_DIR
```

Comment:
Can you instead add this to the from "lms.envs.common import" block towards the top of the file?